AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -393,6 +400,7 @@ private void AddErrorsToModelState(IdentityResult result)
             }
         }
 
+        //TODO: either extract this into a provider pattern or somehow decide which of these conditionals are invoked by which caller and get rid of this method.

```

Comment:
@MisterJames.  I did not know about the base method. I'll check it out and make it part of the rebase.